deploy/min/config.min.yaml

```diff
@@ -100,4 +100,4 @@ component:
       worker: core
       endpoint: mastodon
       parameters:
-        kafka_topic: activitypub_events
+        relay_url_list: ["https://relay.fedi.buzz/instance/mastodon.social"]
```

Comment:
Why do users need to care about this parameter instead of us writing it directly in the code?

Reply:
Allowing the operator to choose which relays to subscribe to offers greater flexibility compared to hardcoding these values. By making it a parameter, users can customize their configuration to meet their specific needs, making the system more adaptable.